checks/config.go

```diff
@@ -57,12 +59,24 @@ type Config struct {
 	// []string{".*", "_*"}.  This is a glob that is applied to each path
 	// component of each file.
 	IgnorePatterns []string `yaml:"ignore_patterns"`
+
+	// MaxConcurrent, if not zero, is the maximum number of concurrent processes
```

Comment:
I'm wondering if this should be part of the config instead. Then we can only limit it on mode 'ci'. It'd simplify the code. I'll continue thinking about that and will report back.